01_fib/cuminandpaprika/main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fib(7)
+}
+
+// Print n number of fibonnaci sequence
+// 1, 1, 2, 3, 5, 8, 13
+// Note: we take the absolute value of n to ensure our loop works.
+func fib(n int16) {
```

Comment:
any reason you went with `int16` specifically here?

It's no problem, just curious whether there was a motivation for being explicit

Reply:
Actually, no I think i didn't give too much thought to it. I changed from byte to int16 just so I could handle signed values and I instinctively chose bytes coming from an embedded software development background. Refactored to int64 to handle a larger range of inputs! 

Thanks for the comment!